tests/backends/test_creation.py

```diff
@@ -54,6 +54,15 @@ def test_custom_test_name_with_test_prefix(self):
 @unittest.skipUnless(connection.vendor == 'postgresql', "PostgreSQL-specific tests")
 class PostgreSQLDatabaseCreationTests(SimpleTestCase):
 
+    def _execute_raise_database_already_exists(self, cursor, parameters, keepdb=False):
```

Comment:
Would it be better to put the helper methods near the tests they're used in? (I'm not sure whether or not they'll be reused for other tests or if there are any guidelines about this ordering).